### ISSUE TITLE
Livechat-Widget not accessible

### ISSUE THREAD

**Issue by mrsimpson** [RocketChat/Rocket.Chat#9500]
### Description:
The Livechat-widget does not allow for a proper use by blind visitors
### Server Setup Information:

* Version of Rocket.Chat Server: 0.59.3

### Steps to Reproduce:

1. Set up your screenreader
2. Open a page where the livechat-widget resides
3. Try to find the input using the keyboard
4. Type something in
5. Respond from the server as agent
6. You will only get a notification, that <Name of the agent> has sent a message, but you'll not be able to find the message-container to get the message read out loud.

### Expected behavior:

When using a screenreader, the livechat widget's controls should be reachable via `TAB`

### Actual behavior:

The widget's controls are being left out
